src/main/scala/org/hammerlab/guacamole/commands/jointcaller/VCFOutput.scala

```diff
@@ -79,7 +79,9 @@ object VCFOutput {
     extraHeaderMetadata.foreach(kv => {
       header.addMetaDataLine(new VCFHeaderLine(kv._1, kv._2))
     })
-    header.addMetaDataLine(new VCFHeaderLine("reference", reference.source))
+    if (reference.source.isDefined) {
```

Comment:
nit: this could also be `reference.source.foreach( header.addMetaDataLine(new VCFHeaderLine("reference", _)) )`

fine with leaving it as is